.circleci/config.yml

```diff
@@ -458,29 +682,48 @@ jobs:
 
 workflows:
   version: 2
-  unittest:
+#  unittest:
```

Comment:
Any reason this is commented out?  You could either add a TODO describing when to uncomment it, or uncomment it now.

Reply:
Yeah, that was a testing change (so it didn't trigger all the builds on push to WIP pr).

I will uncomment it before merging into master :)